unity/Assets/Maroon/scenes/experiments/PointWave/scripts/PointWavePoolHandler.cs

```diff
@@ -201,6 +201,7 @@ public void RemoveSource(PointWaveSource source)
 
     public void ResetObject()
     {
+        ResetWholeObject();
```

Comment:
Should we move the code from the ResetWholeObject method to ResetObject instead of calling ResetWholeObject, or is ResetWholeObject used somewhere?